packages/compass-e2e-tests/helpers/commands/set-ace-value.js

```diff
@@ -25,11 +24,23 @@ module.exports = function (app) {
       return focused;
     });
 
-    await client.keys([META, 'a']);
-    await client.keys([META]); // meta a second time to release it
-    await client.keys(['Backspace']);
-    app.electron.clipboard.writeText(value, 'clipboard');
-    await client.keys([META, 'v']);
-    await client.keys([META]); // meta a second time to release it
+    const META = process.platform === 'darwin' ? 'Meta' : 'Control';
+
+    await browser.keys([META, 'a']);
+    await browser.keys([META]); // meta a second time to release it
+    await browser.keys(['Backspace']);
+
+    await clipboard.write(value);
+
+    // For whatever reason it is shift-insert and not cmd-v  ¯\_(ツ)_/¯
+    // https://twitter.com/webdriverio/status/812034986341789696?lang=en
+    // https://bugs.chromium.org/p/chromedriver/issues/detail?id=30
```

Comment:
Shift+Insert is indeed a combo that will also insert text and IIRC it should work for other platforms too. Worth leaving a comment that there is a problem with Ctrl+V but maybe we can remove the if here and try using it everywhere?